yt/yt/server/query_tracker/yql_engine.cpp

```diff
@@ -133,6 +143,7 @@ class TYqlQueryHandler
     const NApi::NNative::IConnectionPtr Connection_;
     const TYqlSettingsPtr Settings_;
     const TString Stage_;
+    const EExecuteMode ExecutionMode_;
```

Comment:
It should be either execute mode or execution mode both in type name and in variable.